apps/api/src/app/topics/e2e/helpers/index.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'chai';
+import { UserSession } from '@novu/testing';
+
+import { GetTopicResponseDto } from '../../dtos';
+import { TopicId, TopicKey, TopicName } from '../../types';
+
+const BASE_PATH = '/v1/topics';
+
+export const addSubscribers = async (
+  session: UserSession,
+  topicKey: TopicKey,
+  subscribers: string[]
+): Promise<void> => {
```

Comment:
As in the Layouts feature, added a helpers file to be able to reuse in the different test suites reusable functions of generic actions.